### DISCUSSION TITLE
Moving "question" issues to Discussions

### DISCUSSION THREAD

**Discussion by s-martin** [MiczFlor/RPi-Jukebox-RFID#1190] [Ideas]
I think it might be a good idea to move issues currently labeled with "question" here to Discussions to the Q&A category.

Advantage: the questions don't get buried in the pile of bug issues and users could find questions easier.

I'm not sure, if moving from issues to Discussions is supported, but I think it would be a good idea.

**Comment by s-martin** [comment]
![Screenshot_20201210-203535](https://user-images.githubusercontent.com/6143783/101820764-59208980-3b27-11eb-9bb2-07a5393b038c.png)

Ok, it's possible.

**Reply by ZyanKLee** [reply]
I would move the big discussion issue to discussions as well
